hypha/apply/projects/views/payment.py

```diff
@@ -84,6 +86,9 @@ def form_valid(self, form):
             and self.request.user.is_finance_level_1
             and self.object.status == APPROVED_BY_FINANCE
         ):
+            self.object.approved_by = self.request.user
```

Comment:
It is not the right place to update approved_by or approved_at because here we are just notifying the finance members to approve the invoice.

It requires adding another condition to check if the status is `Approved by Finance`(without extended).